tool/tsh/tsh.go

```diff
@@ -1981,3 +1992,14 @@ func readClusterFlag(cf *CLIConf, fn envGetter) {
 // envGetter is used to read in the environment. In production "os.Getenv"
 // is used.
 type envGetter func(string) string
+
+// parameter must be yes, no, auto or only
+func validateAgentKeyOption(supplied string) error {
+	switch supplied {
+	case client.AddKeysToAgentOnly, client.AddKeysToAgentAuto, client.AddKeysToAgentYes, client.AddKeysToAgentNo:
```

Comment:
Group these in a single slice - that way it will be easier to manage (incl. the help string above). I'd also move this to the client package - closer to where these have been defined.